.circleci/config.yml

```diff
@@ -559,7 +559,7 @@ workflows:
       - notify-new-app-version:
```

Comment:
The PR LGTM, if not needed you can avoid sending the slack message too

Reply:
We still send it so that people do not freak out by installing it via Testflight and seeing the MVL UI.